certification/pyxis/submit.go

```diff
@@ -0,0 +1,54 @@
+package pyxis
+
+import (
+	"context"
+	"strings"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+func (p *pyxisEngine) SubmitResults(containerImage string) error {
+	ctx := context.Background()
+	projectId := p.ProjectId
+	if projectId == "" {
+		return errors.ErrEmptyProjectID
+	}
+	if strings.HasPrefix(projectId, "ospid-") {
+		projectId = strings.Split(projectId, "-")[1]
+	}
+	project, err := p.getProject(ctx)
+	if err != nil {
+		log.Error(err, "could not retrieve project")
+		return err
+	}
+	oldProject := project
+
+	if project.CertificationStatus == "Started" {
+		project.CertificationStatus = "In Progress"
+	}
+
```

Comment:
we also need to add the `docker config` to the project before updating it. maybe just put a `todo` in for now, since I've added that `flag/env` in my PR?

Reply:
Yes. This is really not meant to be an exhaustive patch. It will need to be extended for all of the requirements, but I'm hoping this starts us in the right direction.

Reply:
Yeah, it totally is the right direction and we can add to it. Since the requirements are across a bunch of issues and in documents, `TODO`'s are just easier for me to look and see what is missing.